src/gcp/runtimeconfig.ts

```diff
@@ -1,26 +1,22 @@
-"use strict";
+import * as _ from "lodash";
 
-const { runtimeconfigOrigin } = require("../api");
-const { Client } = require("../apiv2");
-
-const { logger } = require("../logger");
-var _ = require("lodash");
+import { runtimeconfigOrigin } from "../api";
+import { Client } from "../apiv2";
+import { logger } from "../logger";
 
 const API_VERSION = "v1beta1";
 const apiClient = new Client({ urlPrefix: runtimeconfigOrigin, apiVersion: API_VERSION });
 
-function _listConfigs(projectId) {
+function listConfigs(projectId: string): Promise<any> {
   return apiClient
-    .get(`/projects/${projectId}/configs`, {
+    .get<{ configs: any }>(`/projects/${projectId}/configs`, {
       retryCodes: [500, 503],
     })
-    .then(function (resp) {
-      return resp.body.configs;
-    });
+    .then((resp) => resp.body.configs);
 }
 
-function _createConfig(projectId, configId) {
-  var path = _.join(["projects", projectId, "configs"], "/");
+function createConfig(projectId: string, configId: string): Promise<any> {
+  const path = _.join(["projects", projectId, "configs"], "/");
```

Comment:
prefer ["projects", projectId, "configs"].join("/")

Reply:
ack - will do a pass for more of these in the near future